frontend/src/global_styles/content/_toast.sass

```diff
@@ -173,6 +173,8 @@ $nm-upload-box-padding: rem-calc(15) rem-calc(25)
   & p, & div, & ul>li, & .button
     font-size: $nm-font-size
 
+  &.-with-bottom-spacing
+    margin-bottom: 2rem
```

Comment:
@HDinger I would like your opinion on this. Can we do it like this?

Reply:
I agree. We need to use it here. So it is not a toast in general, but the same UI component.

![image](https://user-images.githubusercontent.com/480983/177738700-f404aec1-27cd-4f2a-a96f-74f263f18a60.png)

Reply:
And why isn't that a normal toast that floats at the top of the page?

Reply:
Let me add to that discussion: If you search our code base for `toast -info` you find a couple of places where we are using these boxes (that were renamed to toast), as wrappers for highlighted content. That was a very normal use case and I argue it still is. These are use cases that are unlike flash messages.

Reply:
So, I assume it is okay to add the spacing that way?